SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+Eclipse OpenJ9 follows the [Eclipse Vulnerability Reporting Policy](https://www.eclipse.org/security/policy.php). Vulnerabilities are tracked by the Eclipse OpenJ9 project leads, or by the Eclipse security team in cooperation with the OpenJ9 project leads. Fixing vulnerabilities is taken care of by the OpenJ9 project committers.
+
+## Supported Versions
+
+Eclipse OpenJ9 supports security updates only in upcoming releases.
```

Comment:
```suggestion
Eclipse OpenJ9 supports security updates only in upcoming OpenJ9 releases.  

Since OpenJ9 currently uses a single stream to support the Long Term Support (LTS) releases of Java (8, 11 and soon 17) as well as the feature releases, fixes to the single OpenJ9 release will be available in each of the next Java releases.
```